src/api/SFA.DAS.Provider.Events.sln

```diff
@@ -27,8 +27,6 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SFA.DAS.Provider.Events.Api
 EndProject

```

Comment:
remove this section as both of these files have been deleted

Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Build", "Build", "{DAFC5414-B8A0-433A-B50E-367087ABD671}"
	ProjectSection(SolutionItems) = preProject
		Build.fsx = Build.fsx
		RunBuild.bat = RunBuild.bat
	EndProjectSection
EndProject

Reply:
Section removed